incubator/logstash/values.yaml

```diff
@@ -123,6 +123,8 @@ readinessProbe:
   # failureThreshold: 6
   # successThreshold: 1
 
+# command: logstash-plugin install logstash-filter-uuid; /usr/local/bin/docker-entrypoint
```

Comment:
Is installing plugins the only reason this will be needed?
I am not a fan of installing plugins into containers on bootup.
It's best to build a container with those plugins already installed and use that.

Reply:
@rendhalver To be transparent, yes that is my only need for it, but others may find it useful in different ways. I agree that what you suggested is a better option, so feel free to close the PR if you disagree with the change. Otherwise I implemented it so the command was essentially "turned off" by default, and is simply there for convenience.